<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Latr</string>
  </property>
  <property name="windowIcon">
   <iconset theme="scroll.icns"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>531</width>
      <height>221</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPandoc">
     <attribute name="title">
      <string>Pandoc</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select the output location (File will be .docx)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOutputPandoc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblOutputPandoc">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>75</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bibliography</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBib">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBib">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>include bib?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>116</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>bib style</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCSL">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCSL">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>115</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose cls</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGeneratePandoc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLatexdiff">
     <attribute name="title">
      <string>Latexdiff</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>511</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select a previous LaTex source</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrevious">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPrevious">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>491</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select the output location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerateDiff">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDiffOutput">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblOutputDiff">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select a LaTex source file</string>
    </property>
   </widget>
   <widget class="QLabel" name="sourceLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLatexSouorce">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuPreferences"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBlah">
   <property name="text">
    <string>Blah</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
